packages/common-scripts/index.d.ts

```diff
@@ -1,7 +1,9 @@
-import { TransactionSkeleton } from "@ckb-lumos/types"
+import { TransactionSkeleton } from "@ckb-lumos/helpers"
 
 export type Address = string
 export type Config = any // TODO: define this type later
+export type FullCell = any
```

Comment:
Let's take some time to define FullCell here, Config might need more thinking, hence it is a pending item, but FullCell can be defined now.

Reply:
updated